packages/metal-dom/src/events.js

```diff
@@ -2,6 +2,54 @@
 
 import { registerCustomEvent } from './dom';
 import features from './features';
+import * as KEYMAP from './keyConstants';
+
+const REGEX_EVENT = /^(key+(down|up|press)):((,?[a-z])+)/;
+
+/**
+ * Returns the relative keyCode from each key name in an array.
+ * @param {Array<string>} keyNames An array with the key names.
+ */
+function convertKeynamesToKeyCode_(keyNames) {
+  let keys = {};
+
+  keyNames.forEach((value, index) => {
+    keys[KEYMAP[value.toUpperCase()]]= true;
+  });
+
+  return keys;
+}
+
+/**
+ * Creates an event configuration object to deail with keyboard events.
+ * @param {!Array} MatchedRegexInfo The information extracted from the regex
+ * that has matched with the parameterized event name.
+ * @param {object} The custom keyboard event configuration that has all the
+ * necessary informations for the handler function.
+ */
+function createCustomKeyboardEventConfig(MatchedRegexInfo) {
```

Comment:
Uppercase variables are usually only used for global constants or constructor functions. This is just an array, so the name should be `matchedRegexInfo` instead.